lib/traject/vatican_iiif_config.rb

```diff
@@ -24,6 +27,189 @@
+  to_field 'manuscript_extent_tesim', extract_xml('msPart/physDesc/extent', nil) #	Numero fogli	Extent
+  to_field 'manuscript_content_tesim', extract_xml('msPart/msContents/p', nil) #	Contenuto	Content
+  to_field 'manuscript_overview_tesim', extract_xml('msPart/msContents/overview/p/@value', nil) #		Nota generale	Overview
+  to_field 'manuscript_collation_tesim', extract_xml('msPart/physDesc/collation/p', nil) #	Collazione	Collation
+  to_field 'manuscript_layout_tesim', extract_xml('msPart/physDesc/layout/p', nil) #	Impaginazione	Layout
+  to_field 'manuscript_foliation_tesim', extract_xml('msPart/physDesc/foliation/p', nil) #	Foliazione	Foliation
+  to_field 'manuscript_writing_tesim', extract_xml('msPart/head/scriptTerm', nil) #	Scrittura	Writing
+  to_field 'manuscript_writing_note_tesim', extract_xml('msPart/physDesc/msWriting/p', nil) #	Scrittura - Nota	Writing - Note
+  to_field 'manuscript_music_notation_tesim', extract_xml('msPart/physDesc/musicNotation/p', nil) #	Notazione musicale	Music notation
+  to_field 'manuscript_punctuation_tesim', extract_xml('msPart/physDesc/punctuation/p', nil) #	Punteggiatura	Punctuation
+  to_field 'manuscript_decoration_tesim', extract_xml('msPart/physDesc/decoration/p', nil) #	Decorazione	Decoration
+  to_field 'manuscript_decoration_note_tesim', extract_xml('msPart/physDesc/decoration/decoNote/p', nil) #	Decorazione - Nota	Decoration - Note
+  to_field 'manuscript_binding_tesim', extract_xml('msPart/physDesc/bindingDesc/binding/p', nil) #	Legatura	Binding
+  to_field 'manuscript_binding_note_tesim', extract_xml('msPart/physDesc/bindingDesc/binding/decoNote/p', nil) #	Legatura -Nota	Binding - Note
+  to_field 'manuscript_additions_tesim', extract_xml('msPart/physDesc/additions/p', nil) #	Allegati	Additions
+  to_field 'manuscript_condition_tesim', extract_xml('msPart/physDesc/condition/p', nil) #	Stato di conservazione	Condition
+  to_field 'manuscript_signatures_tesim', extract_xml('msPart/physDesc/signatures', nil) #	Segnature di fascicoli	Signatures
+  to_field 'manuscript_catchwords_tesim', extract_xml('msPart/physDesc/catchwords', nil) #	Verba reclamantia	Catchwords
+  to_field 'manuscript_palimpsest_tesim', extract_xml('msPart/physDesc/palimpsest/p', nil) #	Palinsesto	Palimpsest
+  to_field 'manuscript_physical_description_tesim', extract_xml('msPart/physDesc/p', nil) #	Descrizione fisica	Physical description
+  to_field 'manuscript_heraldry_tesim', extract_xml('msPart/head/heraldry', nil) #	Stemma	Heraldry
+  to_field 'manuscript_seal_tesim', extract_xml('msPart/head/seal', nil) #	Sigillo	Seal
+  to_field 'manuscript_format_tesim', extract_xml('msPart/head/format', nil) #	Formato	Format
+  to_field 'manuscript_watermarks_tesim', extract_xml('msPart/head/watermarks/p', nil) #	Filigrane	Watermarks
+  to_field 'manuscript_motto_tesim', extract_xml('msPart/head/motto', nil) #	Motto	Motto
+  to_field 'manuscript_locus_tesim', extract_xml('msPart/msContents/msItem/locus', nil) #	Locus	Locus
```

Comment:
I've noticed that from here to the end, I've mistakenly indexed data marked as section-level. I'm going to verify that it doesn't appear at the manuscript level and then remove it.. hopefully it'll be useful for section-level work, at least.

Reply:
Ok, some of these fields do appear on `n="y"` descriptions. I guess we should consult with @ggeisler about them. It might be easiest on us if we can handle the `n="y"` and `n="n"` sections identically, rather than trying to assign fields to one or the other?